code/modules/mob/living/simple_animal/hostile/megafauna/chaos_marine.dm

```diff
@@ -0,0 +1,510 @@
+/mob/living/simple_animal/hostile/megafauna/chaos_marine
+	name = "chaos marine"
+	desc = "A servant of dark gods, chaos marine is always in state of eternal battle."
+	health = 2000
+	maxHealth = 2000
+	icon_state = "chaos_marine"
+	icon_living = "chaos_marine"
+	icon = 'icons/Fulpicons/megafauna.dmi'
+	faction = list("chaos")
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	light_color = "#FFFFFF"
+	movement_type = GROUND
+	speak_emote = list("roars")
+	speed = 6
+	light_range = 6
+	move_to_delay = 4
+	projectiletype = /obj/projectile/bullet/chaos_bomb
+	projectilesound = 'sound/weapons/gun/l6/shot.ogg'
+	ranged = TRUE
+	ranged_cooldown_time = 20
+	vision_range = 10
+	damage_coeff = list(BRUTE = 1, BURN = 0.5, TOX = 0.5, CLONE = 0.5, STAMINA = 0, OXY = 0.5)
+	loot = list(/obj/item/nullrod/scythe/talking/chainsword/chaos, /obj/item/clothing/suit/space/hardsuit/ert/paranormal/berserker/chaos)
+	crusher_loot = list(/obj/item/nullrod/scythe/talking/chainsword/chaos, /obj/item/clothing/suit/space/hardsuit/ert/paranormal/berserker/chaos)
+	wander = FALSE
+	del_on_death = TRUE
+	blood_volume = BLOOD_VOLUME_NORMAL
+	gps_name = "Chaotic Signal"
+	deathmessage = "falls to the ground, decaying into glowing particles."
+	deathsound = "sound/magic/curse.ogg"
+	footstep_type = FOOTSTEP_MOB_HEAVY
+	attack_action_types = list(/datum/action/innate/megafauna_attack/blood_dash,
+							   /datum/action/innate/megafauna_attack/teleport_b,
+							   /datum/action/innate/megafauna_attack/runic_blast,
+							   /datum/action/innate/megafauna_attack/infernal_summon,
+							   /datum/action/innate/megafauna_attack/blast,
+							   /datum/action/innate/megafauna_attack/rapid_fire)
+	move_force = MOVE_FORCE_NORMAL
+	var/turf/starting
+	var/obj/item/nullrod/scythe/talking/chainsword/chaos/mob/weapon
+	var/charging = FALSE
+	var/dash_cooldown = 6 SECONDS
+	var/runic_blast_cooldown = 14 SECONDS
+	var/teleport_cooldown = 6 SECONDS
+	var/infernal_summon_cooldown = 30 SECONDS
+	var/dash_mod = 1
+	var/dash_num = 3
+	var/newcolor = rgb(149, 10, 10)
+
+/obj/item/nullrod/scythe/talking/chainsword/chaos
+	name = "chainsaw sword"
+	desc = "Blood for the blood god!"
+	throwforce = 25
+	block_chance = 35
+	armour_penetration = 50
+	wound_bonus = 15
+	bare_wound_bonus = 30
+	sharpness = SHARP_EDGED
+
+/obj/item/nullrod/scythe/talking/chainsword/chaos/mob
+	wound_bonus = 0
```

Comment:
might wanna make this be a wound_bonus = CANT_WOUND weapon, since none of the other megafauna are meant to inflict RNG-based wounds on you